sdk/samples/storage/blobs/README.md

```diff
@@ -52,11 +52,11 @@ Azure SDK for Embedded C is licensed under the [MIT][azure_sdk_for_c_license] li
 <!-- LINKS -->
 [azure_sdk_for_c_contributing]: https://github.com/Azure/azure-sdk-for-c/blob/main/CONTRIBUTING.md
 [azure_sdk_for_c_license]: https://github.com/Azure/azure-sdk-for-c/blob/main/LICENSE
-[BLOB_SDK_README]: https://github.com/Azure/azure-sdk-for-c/tree/main/sdk/docs/storage
-[SDK_README_CONTRIBUTING]:https://github.com/Azure/azure-sdk-for-c/tree/main/sdk/docs/storage#contributing
-[SDK_README_GETTING_STARTED]: https://github.com/Azure/azure-sdk-for-c/tree/main/sdk/docs/storage#getting-started
-[SDK_README_KEY_CONCEPTS]: https://github.com/Azure/azure-sdk-for-c/tree/main/sdk/docs/storage#key-concepts
-[samples_basic]: https://github.com/Azure/azure-sdk-for-c/blob/main/sdk/samples/storage/blobs/src/blobs_client_sample.c
```

Comment:
Interesting that the failed leg only detected two invalid links, when there were some more:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1187023&view=logs&j=a129effc-2dd1-54d1-fb5a-ad7bdc0e851d&t=617ceecd-bf48-5ce3-83e2-7468b1632a13
> 'file:///D:/a/_work/1/s/sdk/samples/storage/blobs/README.md' has 2 broken link(s):
  https://github.com/Azure/azure-sdk-for-c/tree/main/sdk/docs/storage
  https://github.com/Azure/azure-sdk-for-c/blob/main/sdk/samples/storage/blobs/src/blobs_client_sample.c